src/DependencyInjection/Configuration.php

```diff
@@ -293,8 +293,18 @@ private function addFormSection($nodeBuilder)
                             ->scalarNode('seo_metadata')->defaultNull()->end()
                         ->end()
                     ->end()
+                    ->arrayNode('options')
+                        ->addDefaultsIfNotSet()
+                        ->children()
+                            ->enumNode('advanced')
```

Comment:
With the info it sounds ok, but the isn't really speaking else.
What we do with that key-value stuff is arbitrary metadata, why not going in that direction?